js/tji-donate.js

```diff
@@ -20,10 +20,22 @@ jQuery(document).ready(function($){
 	});
 
 	// Verify form details upon submit
-	$('.tji-donation-submit').click(function() {
-		event.preventDefault();
+	$('.tji-donation-submit').click(function(e) {
+		e.preventDefault();
+
+		var error = false;
+
+		$('input.required').each(function() {
+			if (this.value == "") {
+				$('label[for="' + this.id + '"]').children('.donation-form__error').css("display", "inline-block");
+				error = true;
+				return false;
+			} else {
+				$('label[for="' + this.id + '"]').children('.donation-form__error').css("display", "none");
+			}
+		});
 		
-		if ($('.selected').length) {
+		if (error == false && $('.selected').length) {
```

Comment:
so I think we should check to see that the donation amount is a number and greater than 0 if someone puts a custom amount, right?